app/utils/lava_log_parser.py

```diff
@@ -87,11 +114,19 @@ def run(log, boot, txt, html):
         timestamp = "<span class=\"timestamp\">{}  </span>".format(raw_ts)
 
         fmt = formats.get(level)
+        
+        if(isinstance(msg, list)):
```

Comment:
Does this actually happen?  Can a message from the LAVA log be a list?

Reply:
yes, that first case that broke in staging was about this..and not related to any specific char.

Reply:
> - {"dt": "2019-08-29T19:32:41.124792", "lvl": "debug", "msg": ["-+\\[ cut here \\]-+\\s+(.*\\s+-+\\[ end trace (\\w*) \\]-+)", "(Unhandled fault.*)\\r\\n", "Kernel panic - (.*) end Kernel panic", "Stack:\\s+(.*\\s+-+\\[ end trace (\\w*) \\]-+)", "/ #", "Login timed out", "Login incorrect"]}

probably is the only case that we have msg as list

Reply:
OK that looks like a new thing in LAVA that wasn't there when the script was created.  It would be better to keep the code that joins the strings together as a separate commit I think, as it's not really about highlighting kernel messages.